de_lon8/Kareem/Mini_Project_1/Week 1 simple.py

```diff
@@ -173,43 +176,80 @@
                 print (courier)
 
 
-    # if choice == 2:
-    #     while True:
-    #         print ('')
-    #         print ('')
-    #         print ('ORDERS MENU')
-    #         print ('0. Return to the MAIN MENU.')
-    #         print ('1. To PRINT all orders')
-    #         print ('2. To CREATE a new order') 
-    #         print ('3. To UPDATE an order')
-    #         print ('4. To DELETE an order')
-    #         print('') 
-
-    #         orders_menu_choice = int(input(''))
-
-    #         if orders_menu_choice == 0:
-    #             break
+    if choice == 2:
+        while True:
+            print ('')
+            print ('')
+            print ('ORDERS MENU')
+            print ('0. Return to the MAIN MENU.')
+            print ('1. To PRINT all orders')
+            print ('2. To CREATE a new order') 
+            print ('3. To UPDATE STATUS')
+            print ('4. To UPDATE an order')
+            print ('5. To DELETE an order') 
 
-    #         elif orders_menu_choice == 1:
-    #             print(orders)
+            orders_menu_choice = int(input(''))
 
-    #         elif orders_menu_choice == 2:
-    #             order_name = input('Please enter the customer name: ')
-    #             order_address = (input('Enter the address number: '))
-    #             order_phone_number = input('Please enter the phone number: ')
-    #             order_status = 'PREPARING'
+            #Return to previous menu
+            if orders_menu_choice == 0:
+                break
+            
+            #Prints all orders
+            elif orders_menu_choice == 1:
+                print(orders)
+
+            #Adding a new order
+            elif orders_menu_choice == 2:
+                order_name = input('Please enter the customer name: ')
+                order_address = (input('Enter the address number: '))
+                order_phone_number = input('Please enter the phone number: ')
+                order_status = 'PREPARING'
                 
-    #             single_order = {'Name': order_name, 'Address' : order_address, 'Phone Number' : order_phone_number, 'Status' : order_status}
+                single_order = {'Name': order_name, 'Address' : order_address, 'Phone Number' : order_phone_number, 'Status' : order_status}
 
-    #             orders.append(single_order)
+                orders.append(single_order)
 
-    #         elif orders_menu_choice == 3:
-    #             y = json.loads(orders)
-    #             print(y)
+            #Updating existing orderstatus
+            elif orders_menu_choice == 3:
+                order_number = 0
+                for i in orders:
```

Comment:
Could we do something like:
```Python
for order in orders:
    ...
print (order_number, ':', order)    
```
Does it read better, perhaps?

Reply:
yes it does actually!!!!